packages/stencil/components/pagination/src/components/osds-pagination/osds-pagination.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { newSpecPage, SpecPage } from '@stencil/core/testing';
+import { OsdsPagination } from './osds-pagination';
+import { OdsComponentAttributes2StringAttributes, OdsPaginationAttributes, odsPaginationDefaultAttributes } from '@ovhcloud/ods-core';
+
+import { OdsCreateAttributes, OdsMockNativeMethod, OdsStringAttributes2Str, odsPaginationBaseAttributes } from '@ovhcloud/ods-testing';
+
+describe('spec:osds-pagination', () => {
```

Comment:
missing tests:
- slot ('content')
- event odsCurrentChange
- some uncovered lines described inside the component in comments
- a change on current should trigger an event (onCurrentChange)
- onKeyPress should call the controller